FirebaseAppCheck/Sources/DebugProvider/FIRAppCheckDebugProvider.m

```diff
@@ -85,8 +85,6 @@ - (void)getTokenWithCompletion:(void (^)(FIRAppCheckToken *_Nullable token,
   [self.debugProvider getTokenWithCompletion:^(GACAppCheckToken *_Nullable internalToken,
                                                NSError *_Nullable error) {
     if (error) {
-      FIRAppCheckDebugLog(kGACLoggerAppCheckMessageDebugProviderFailedExchange,
```

Comment:
Is this delete intentional?

Reply:
It is intentional but thank you for double checking!

Context: The message with code `GACLoggerAppCheckMessageDebugProviderFailedExchange` is already logged in [`GACAppCheckDebugProvider`](https://github.com/firebase/firebase-ios-sdk/blob/05f0637c9eed908f5051b76b8a00e9f66089a97e/AppCheck/Sources/DebugProvider/GACAppCheckDebugProvider.m#L115-L117). I had already refactored `kGACLoggerAppCheckMessageDebugProviderFailedExchange` in PR #11570 but it didn't trigger FirebaseAppCheck's CI checks so I didn't catch it until now.